### PR TITLE
[cli] Support "http:" scheme in `vc bisect`

### PR DESCRIPTION
**Pull request by MatthewStanciu**
`vc bisect` currently prepends `https://` to a passed-in url if it doesn't begin with https—which means that if someone passes in a url that begins with `http://`, it'll turn the url into `https://http://url.com`. This PR fixes this.

#### Tests

- [ ] The code changed/added as part of this PR has been covered with tests
- [ ] All tests pass locally with `yarn test-unit`

#### Code Review

- [ ] This PR has a concise title and thorough description useful to a reviewer
- [ ] Issue from task tracker has a link to this PR

### CODE REVIEW

File: packages/cli/src/commands/bisect/index.ts [hunk 1]
```diff
@@ -370,6 +366,16 @@ export default async function main(client: Client): Promise<number> {
...
 }
 
+function hasScheme(url: string): Boolean {
+  return url.startsWith('http://') || url.startsWith('https://');
+}
+
+function validateUrl(url: string): string {
+  if (!hasScheme(url)) {
+    return `https://${url}`;
+  } else return url;
```
Comment (styfle) [inline packages/cli/src/commands/bisect/index.ts]:
```suggestion
  }
  return url;
```

No `else` needed here because of the early return

Comment (styfle) [reply packages/cli/src/commands/bisect/index.ts]: Actually, you can do it with a single return

```ts
return hasScheme(url) ? url : `https://${url}`
```

File: packages/cli/src/commands/bisect/index.ts [hunk 2]
```diff
@@ -370,6 +366,16 @@ export default async function main(client: Client): Promise<number> {
   return 0;
 }
 
+function hasScheme(url: string): Boolean {
+  return url.startsWith('http://') || url.startsWith('https://');
+}
+
+function validateUrl(url: string): string {
```
Comment (styfle) [inline packages/cli/src/commands/bisect/index.ts]: How about `normalizeURL` since usually validate implies throwing an error if its invalid

**Review by EndangeredMassa — APPROVED** [review]

**Review by TooTallNate — APPROVED** [review]

File: packages/cli/src/commands/bisect/index.ts [hunk 3]
```diff
@@ -370,6 +366,14 @@ export default async function main(client: Client): Promise<number> {
   return 0;
 }
 
+function hasScheme(url: string): Boolean {
+  return url.startsWith('http://') || url.startsWith('https://');
+}
+
+function normalizeURL(url: string): string {
+  return hasScheme(url) ? url : `https://${url}`;
+}
```
Comment (TooTallNate) [inline packages/cli/src/commands/bisect/index.ts]: For the purposes of testing this PR, I would recommend moving this into a file in `src/utils/commands/bisect` dir and adding some unit tests with various input values.

Comment (MatthewStanciu) [reply packages/cli/src/commands/bisect/index.ts]: 👍 Just added tests

Comment (TooTallNate) [reply packages/cli/src/commands/bisect/index.ts]: Awesome. Thanks!

File: packages/cli/test/unit/util/bisect/normalize-url.test.ts [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+describe('normalize-url', () => {
+  it('should add https to url without scheme', () => {
+    const normalizedUrl = normalizeURL('vercel.com');
+    expect(normalizedUrl).toEqual('https://vercel.com');
+  });
+  it('should not add anything to a url that starts with https', () => {
+    const normalizedUrl = normalizeURL('https://vercel.com');
+    expect(normalizedUrl).toEqual('https://vercel.com');
+  });
+  it('should not add anything to a url that starts iwth http', () => {
```
Comment (TooTallNate) [inline packages/cli/test/unit/util/bisect/normalize-url.test.ts]:
```suggestion
  it('should not add anything to a url that starts with http', () => {
```

**Review by styfle — APPROVED** [review]
Thanks!

**Pull request merged**